packages/babel/test/fixtures/transformation/optimisation.react.constant-elements/inline-elements/expected.js

```diff
@@ -1,6 +1,9 @@
 "use strict";
 
+var _typeofReactElement = typeof Symbol === "function" && Symbol["for"] && Symbol["for"]("react.element") || 60103;
```

Comment:
Kinda unfortunate that we lose our hex speak here. :) Consider this an enhancement request to preserve hex literals by default?